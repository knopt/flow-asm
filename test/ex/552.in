31
1
-3.6613887955498825
-8.549203504286 -3.2055595100938845 -5.070545606854317 -9.925508991859353 -0.577851488705079 7.437214172597212 -1.5848355312535674 2.228446095707067 3.563684880470575 0.6021166607221105 -1.7812816017417337 2.968941258207092 6.568606304094409 -7.1034624627337895 0.2710193010983115 -1.5838043934772053 -6.766130647678967 8.526752813819563 5.833331750693487 -7.296482843597096 9.22627858904508 3.157567809794422 -8.097690907817823 -4.832455036863581 7.907295544897565 -7.696905486655384 3.1609710463671075 -0.7707058257966093 4.8827734183971945 3.804726244215484 -7.4137814740144155 

-3.582982034704612 
