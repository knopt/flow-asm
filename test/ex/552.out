-26.732471149230562 16.359598737648547 -11.898984805389274 -27.70141753252172 33.64755695798177 36.78348578048603 -34.61806722982695 16.19035271893696 8.452513206387323 -10.24133603632153 -10.50782929525991 20.36135401398864 19.748379570838786 -57.162221857586054 27.271864204180098 -8.37503508651644 -25.74064192969729 64.51994497140038 -4.02832995153438 -55.36983888677996 69.72253217006488 -19.062341840786733 -49.307569067498434 7.122842995875842 54.55247558305793 -64.82995230709128 42.91587852783966 -15.166103473259449 25.582358978962205 -0.142423600407644 -48.48909993633124 
